ts/hooks/useParamSelector.ts

```diff
@@ -227,7 +307,13 @@ export function useIsOutgoingRequest(convoId?: string) {
   );
 }
 
-export function useConversationPropsById(convoId?: string) {
+/**
+ * Note: NOT to be exported:
+ * This selector is too generic and needs to be broken node in individual fields selectors.
```

Comment:
`broken node` ?

Reply:
fixed to be 
` * This selector is too generic and needs to be broken down into individual fields selectors.`